lib/gitlab_access.rb

```diff
@@ -19,7 +19,12 @@ def initialize(repo_path, actor, changes)
 
   def exec
     begin
-      status = api.check_access('git-receive-pack', @repo_name, @actor, @changes)
+      status = api.check_access(
+                                'git-receive-pack', 
```

Comment:
Indent the first parameter one step more than the previous line.<br>Trailing whitespace detected.